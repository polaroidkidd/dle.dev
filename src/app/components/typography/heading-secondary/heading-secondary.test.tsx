import React from "react";
import renderer from "react-test-renderer";
import { HeadingSecondary } from "./index";

test("Render HeadingSecondary with text", () => {
  const tree = renderer.create(<HeadingSecondary text="test heading"/>).toJSON();
  expect(tree).toMatchSnapshot();
});

test("Render HeadingSecondary with empty string", () => {
  const tree = renderer.create(<HeadingSecondary text=""/>).toJSON();
  expect(tree).toMatchSnapshot();
});