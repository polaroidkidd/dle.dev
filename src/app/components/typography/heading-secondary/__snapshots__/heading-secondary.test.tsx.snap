// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render HeadingSecondary with empty string 1`] = `
<h2
  className="heading-secondary"
>
  
</h2>
`;

exports[`Render HeadingSecondary with text 1`] = `
<h2
  className="heading-secondary"
>
  test heading
</h2>
`;
